### DISCUSSION TITLE
希望能在“自定义顶栏”组件中增加“动态列表中单列显示视频”选项

### DISCUSSION THREAD

**Discussion by future93** [the1812/Bilibili-Evolved#4000] [功能建议 / Ideas]
版本：v2.6.2-39-gc96b1880c

如标题，自定义顶栏的功能相当不错，但我个人而言，对“动态列表中的视频以双列显示”这一点不太习惯。
如果能给一个“单列显示”的选项，让视频动态能像原版视频列表，或者像组件改进过的动态列表中的番剧、专栏分页那样“单列全宽”地显示条目就好了。

**Comment by Geeyun-JY3** [comment]
不知道 #3304 算不算拒绝了
不过也还有 #2391

**Reply by future93** [reply]
> 不知道 #3304 算不算拒绝了 不过也还有 #2391

啊这……总之也只好等等看作者表态了，真不行那也只能算了。

**Reply by the1812** [reply]
算重复 #2391, 合理但我不一定做(

**Reply by future93** [reply]
> 算重复 #2391, 合理但我不一定做(

感谢
